doc/create-announcement-article.rb

```diff
@@ -0,0 +1,336 @@
+require 'optparse'
+
+class ArticleGenerator
+  attr_reader :output_file_path
+
+  def initialize(release_date, groonga_version, groonga_previous_version, groonga_org_repository)
+    @groonga_version = groonga_version
+    @groonga_previous_version = groonga_previous_version
+    @groonga_version_in_link = groonga_version.gsub(".", "-")
+    @release_date = release_date
+    @release_date_in_link = release_date.gsub("-", "/")
+  end
+
+  def extract_latest_release_note
+    latest_release_note = File.read(@input_file_path).split(/#{@release_headline_regexp_pattern}/)[1]
+    latest_release_note.gsub(/^\R\R/, "\n").strip
+  end
+
+  def generate
+    File.open(@output_file_path, "w") do |file|
+      article = generate_article
+      file.puts(article)
+    end
+  end
+end
+
+class MarkdownArticleGenerator < ArticleGenerator
+  def extract_latest_release_note
+    super
+  end
+end
+
+class MarkdownEnArticleGenerator < MarkdownArticleGenerator
+  def initialize(release_date, groonga_version, groonga_previous_version, groonga_org_repository)
+    super(release_date, groonga_version, groonga_previous_version, groonga_org_repository)
+    @input_file_path = "./locale/en/markdown/news.md"
+    @release_headline_regexp_pattern = "## Release.+\\d\\d\\d\\d-\\d\\d-\\d\\d.*"
+  end
+
+  def generate_article
+    <<-"ARTICLE"
```

Comment:
We don't need to use `"` explicitly:

```suggestion
    <<-ARTICLE
```